packages/server-bridge/src/attachHttpBridgeToServer.js

```diff
@@ -1,64 +1,72 @@
-import jwt from 'express-jwt';
 import bodyParser from 'body-parser';
 
-import split from 'lodash/split';
 import join from 'lodash/join';
-import assign from 'lodash/assign';
+import split from 'lodash/split';
+import uuid from 'uuid/v4';
 
-import startSessionFactory from './factories/startSessionFactory';
-import handleMessageFactory from './factories/handleMessageFactory';
+import defaultHandleMessageFactory from './factories/handleMessageFactory';
+import defaultReportErrorFactory from './factories/reportErrorFactory';
+import defaultSendResponseFactory from './factories/httpSendResponseFactory';
+import defaultStartSessionFactory from './factories/startSessionFactory';
+import defaultCallHandlersSeriallyFactory from './factories/callHandlersSeriallyFactory';
 import getTxName from './getTxName';
-import callHandlersSeriallyFactory from './factories/callHandlersSeriallyFactory';
 import getConfigWithDefaults from './getConfigWithDefaults';
 
-function assignClientToSession({ session }) {
-  return assign(session, { client: 'http' });
-}
-
 export default function attachHttpBridgeToServer(originalConfig) {
   const config = getConfigWithDefaults(originalConfig);
   const {
-        router,
-        secret,
-        instrumentation,
-        newSessionHandlers,
-        stellarRequest,
-    } = config;
+    router,
+    instrumentation,
+    stellarRequest,
+    reportErrorFactory = defaultReportErrorFactory,
+    startSessionFactory = defaultStartSessionFactory,
+    handleMessageFactory = defaultHandleMessageFactory,
+    callHandlersSeriallyFactory = defaultCallHandlersSeriallyFactory,
+    sendResponseFactory = defaultSendResponseFactory,
+  } = config;
```

Comment:
i do ... but its kinda like just simple configuration code. i make an exception to copy/paste rule because i dont want so much indirection in the configuration code. i want it to be clear how it is created. is it ok?

Reply:
Seems ok.
We can always refactor later